pkg/kudoctl/cmd/init.go

```diff
@@ -110,7 +110,7 @@ func newInitCmd(fs afero.Fs, out io.Writer, errOut io.Writer, client *kube.Clien
 	f.StringVarP(&i.image, "kudo-image", "i", "", "Override KUDO controller image and/or version")
 	f.StringVarP(&i.imagePullPolicy, "kudo-image-pull-policy", "", "Always", "Override KUDO controller image pull policy")
 	f.StringVarP(&i.version, "version", "", "", "Override KUDO controller version of the KUDO image")
-	f.StringVarP(&i.output, "output", "o", "", "Output format")
+	f.StringVarP((*string)(&i.output), "output", "o", "", "Output format")
```

Comment:
could we do rather something like i.output.AsString? 🤔 to not hardcode here expectations about the type alias...

Reply:
I can do:
```
f.StringVarP(i.output.AsStringPtr, "output", "o", "", "Output format")
```